cmd/autoheal/healer.go

```diff
@@ -113,6 +122,20 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 	h.k8sClient = b.k8sClient
 	h.config = config
 
+	// Create the actions memory:
+	// if Throttling is "" or 0h then ignore throttling altogether.
+	if config.Throttling().Interval() != 0*time.Second {
+		actionMemory, _ := memory.NewShortTermMemoryBuilder().
+			Duration(config.Throttling().Interval()).
+			Build()
+
+		h.throttling_disabled = false
+		h.actionMemory = actionMemory
+	} else {
+		h.actionMemory = nil
+		h.throttling_disabled = true
+	}
+
```

Comment:
Note that this is possible due to the behavior of `time.ParseDuration` (see [here](https://play.golang.org/p/0cFkDyo8TCt))